src/GraphBLAS-sharp.Backend/Matrix/SpGeMM/Expand.fs

```diff
@@ -0,0 +1,328 @@
+namespace GraphBLAS.FSharp.Backend.Matrix.SpGeMM
+
+open Brahma.FSharp
+open GraphBLAS.FSharp.Backend.Common
+open GraphBLAS.FSharp.Backend.Common.Sort
+open GraphBLAS.FSharp.Backend.Matrix
+open GraphBLAS.FSharp.Backend.Quotes
+open GraphBLAS.FSharp.Backend.Objects.ClContext
+open GraphBLAS.FSharp.Backend.Objects.ArraysExtensions
+open GraphBLAS.FSharp.Backend.Objects
+open GraphBLAS.FSharp.Backend.Objects.ClCell
+open FSharp.Quotations
+open GraphBLAS.FSharp.Backend.Vector.Sparse
+open GraphBLAS.FSharp.Backend.Objects.ClVector
+open GraphBLAS.FSharp.Backend.Objects.ClMatrix
+
+type Indices = ClArray<int>
```

Comment:
If it is really useful, can it be used everywhere we use indices?

But I'm quite sceptical about type aliases in F# because one still can pass ClArray as a parameter. So, type aliases are not restrictive enough to be useful.